tests/integration.rs

```diff
@@ -6651,6 +6650,12 @@ mod windows {
         fn new() -> Self {
             let disk = WindowsDiskConfig::new(WINDOWS_IMAGE_NAME.to_string());
             let guest = Guest::new(Box::new(disk));
+            #[cfg(target_arch = "aarch64")]
+            let auth = PasswordAuth {
+                username: String::from("cloud"),
+                password: String::from("Cloud123"),
+            };
```

Comment:
You could have created an image with the same credentials, this would have avoided making a difference between aarch64 and x86_64.

Reply:
Yeah, realized that, just too late :) I'd prefer to keep this one, as it's also used elsewhere and thus changing creds would cause a chain reaction. Separating another image just for creds would seem to be not worth it.

Thanks

Reply:
Too bad :(